src/js/cptui-scripts.js

```diff
@@ -103,9 +103,10 @@ postboxes.add_postbox_toggles(pagenow);
 			}
 		}
 
-		var $slugexists = $('#slugexists');
+		var $slugexists          = $('#slugexists');
+		var $overwide_validation = $('#override_validation').is(":checked");
 		if ( typeof cptui_type_data != 'undefined' ) {
-			if (cptui_type_data.existing_post_types.hasOwnProperty(value) && value !== original_slug) {
+			if (cptui_type_data.existing_post_types.hasOwnProperty(value) && value !== original_slug && $overwide_validation == false ) {
```

Comment:
Same thing, but this time I'm commenting in the src version, not the build :D Sorry about that.